tests/editing/totals_and_components/test_totals_and_components.py

```diff
@@ -4904,222 +5009,511 @@ def test_totals_and_components(
             assert (str(exc_info.value)) == str(expected_result)
 
 
-class TestTotalsAndComponentsUATFails:
+# This class is to run a single case of the totals and components method
```

Comment:
I think we can uncomment this by default and just include the first UAT test case in here.  Could you also rename to TestTotalsAndComponentsSingleTest